generate-hash-list.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+tmpfile="$(mktemp)"
+tmpold="$(mktemp)"
+
+git clone https://github.com/minetest/minetest
```

Comment:
```suggestion
git clone https://github.com/minetest/minetest --filter=blob:none --no-checkout
```
Skip all blob content to cut out some like 90% of downloaded data.